### ISSUE TITLE
hello example not working with latest image

### ISSUE THREAD

**Issue by csantanapr** [ibm-functions/runtime-kotlin#10]
I tried the hello example with latest image and I get an error 
**Failed to find specified class: MainKt in provided jar file**
```
🚀  $ cat hello.kt
```
```kotlin
data class User (
    val name: String
)
data class Hello (
    val greeting: String
)
fun main(user: User) : Hello {
    val hello = Hello("Hello " + user.name + "!")
    return hello
}
```
```bash
💩  $ ./build.sh
+ kotlinc hello.kt -d myAction.jar


⛄  $ ./deploy.sh
+ bx wsk action update kotlin myAction.jar --web true --docker ibmfunctions/action-kotlin
ok: updated action kotlin

🔥  $ ./deploy.sh
+ bx wsk action update kotlin myAction.jar --web true --docker ibmfunctions/action-kotlin
ok: updated action kotlin


👽  $ ./test.sh
+ bx wsk action invoke kotlin -b -p name Carlos
ok: invoked /_/kotlin with id bfd6377e805f47ce96377e805f77ced6
{
    "activationId": "bfd6377e805f47ce96377e805f77ced6",
...
    "response": {
        "result": {
            "error": "Failed to find specified class: MainKt in provided jar file"
        },
        "status": "action developer error",
        "success": false
    },

}
```

**Comment by csantanapr** [comment]
I think I got it working, change to use defaults. but we should update the README to make it more clear.

name file `Main.kt` so ClassName is `Mainkt`
doesn't specified package
```
💋  $ cat Main.kt
```
```kotlin
data class User (
    val name: String
)
data class Hello (
    val greeting: String
)
fun main(user: User) : Hello {
    val hello = Hello("Hello " + user.name + "!")
    return hello
}
```
```bash
👽  $ ./build.sh
+ kotlinc Main.kt -d myAction.jar

🍰  $ ./deploy.sh
+ bx wsk action update kotlin myAction.jar --web true --docker ibmfunctions/action-kotlin
ok: updated action kotlin

😄  $ ./test.sh
+ bx wsk action invoke kotlin -b -r -p name Carlos
{
    "greeting": "Hello Carlos!"
}
```

**Comment by seabaylea** [comment]
Yeah - that was the problem. I've opened PR #11 to try to make the steps clearer.

**Comment by csantanapr** [comment]
Closed by #11
